core/src/main/java/org/mini2dx/breakout/Ball.java

```diff
@@ -0,0 +1,56 @@
+package org.mini2dx.breakout;
+
+import com.badlogic.gdx.graphics.Texture;
+import org.mini2Dx.core.engine.geom.CollisionBox;
+import org.mini2Dx.core.graphics.Graphics;
+import org.mini2Dx.core.graphics.Sprite;
+
+public class Ball {
+
+    @SuppressWarnings("FieldCanBeLocal")
+    private final float acceleration = 180;
+    private CollisionBox collisionBox;
+    private Sprite ballSprite;
+    private int verticalMovementSign = 1, horizontalMovementSign = 1;
+    public Ball(){
+        Texture ballTexture = new Texture("ballBlue.png");
```

Comment:
"ballBlue.png" would be considered a magic string. It should be moved to a static final variable at the top of the class.